tezos/new_context/src/working_tree/serializer.rs

```diff
@@ -75,100 +83,110 @@ pub struct KeyNodeDescriptor {
 // Must fit in 1 byte
 assert_eq_size!(KeyNodeDescriptor, u8);
 
-pub fn serialize_entry(
-    entry: &Entry,
+fn serialize_tree(
+    tree: &[(StringId, NodeId)],
```

Comment:
lets start using "directory" for anything that we know is a directory and not a value

Reply:
because "tree" could be a directory or a value

Reply:
I renamed it to `serialize_directory`. I am planning to rename everything correctly in another PR, alongside a better documentation